apache/zeppelin
# ZEPPELIN-531 ] Job management - Filter and Run/stop controller

### What is this PR for?
Job manger front side feature

sort by notebook name (asc/desc),
search for notebook(filtering)
filter by interpreter group.
run / stop to notebook.

It is created with the following additional functions PR.
(E. G., Filter and sort)

this PR is divided from the #921

### What type of PR is it?
Improvement

### Todos
- [x] sort by notebook name (asc/desc),
- [x] search for notebook(filtering)
- [x] filter by interpreter group.
- [x] run / stop to notebook.

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-531
https://issues.apache.org/jira/browse/ZEPPELIN-965
https://issues.apache.org/jira/browse/ZEPPELIN-966
https://issues.apache.org/jira/browse/ZEPPELIN-967
https://issues.apache.org/jira/browse/ZEPPELIN-968
https://issues.apache.org/jira/browse/ZEPPELIN-969
https://issues.apache.org/jira/browse/ZEPPELIN-970

#### Screen Shots
![cap 2016-07-08 16-52-00-956](https://cloud.githubusercontent.com/assets/10525473/16680817/53550fda-452c-11e6-8b32-d6a598202532.png)



<a href="https://cloud.githubusercontent.com/assets/10525473/16680800/336d106e-452c-11e6-80a8-8eed2e8cbc51.gif"> ScreenShots</a>
and working video (animated gif)


### How should this be tested?
1. create and running, modified notebook and paragraph.
1. check to cron or normal notebook status in job manager.
1. use to filter featrue in jobmanagement page
### Screenshots (if appropriate)

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no
